### ISSUE TITLE
Vaddio AV Bridge MATRIX PRO

### ISSUE THREAD

**Issue by mknerr** [bitfocus/companion-module-requests#691]
Requesting support for the [Vaddio AV Bridge MATRIX PRO](https://www.legrandav.com/products/av_to_usb_bridges/production_switcher/av-bridge-matrix-pro/999-8230-000#sort=relevancy&numberOfResults=20). The API is well documented in the attached manual.

This is a audio/video mixing/switching and streaming device, but it lacks any (meaningful) physical control, so even basic functions would be helpful. Switching, muting, PIP, Enabling streams, etc. Thank you for considering!

[411-0020-30_Rev_E_AV_Bridge_MATRIX_PRO_Complete_Manual.pdf](https://github.com/bitfocus/companion-module-requests/files/7859489/411-0020-30_Rev_E_AV_Bridge_MATRIX_PRO_Complete_Manual.pdf)

**Comment by krocheck** [comment]
I wrote a module for [Vaddio PTZs](https://github.com/bitfocus/companion-module-vaddio-ptz) a while back.  My recollection is that Vaddio uses the same telnet 'protocol' for all their stuff.  Should be relatively straightforward for a savvy person to adapt that code as a base for this.  I don't have availability for a while to put this on my plate.

**Comment by mknerr** [comment]
Thanks for the reply. I did see that, and thank you for the work put in! It looks very thorough. I don't have any of their cameras yet, but I have my eye out, so that will probably come in handy.

**Comment by krocheck** [comment]
I was noting that for other people who may have the time to pick this request up ... the foundation for this module should already be laid ;)

**Comment by mknerr** [comment]
@krocheck Yes, that was my impression. I was just thanking you for laying that foundation. 
I did poke around a bit in your module, and there are some things I could easily copy and adapt, but a lot was still over my head. The good news is, it should be much less complicated than the camera control, since most of what's needed will be basic on/off functions.

**Comment by josephdadams** [comment]
Duplicate of #279 — consolidating the discussion there so it stays in one place.
